docs/development/setup-vagrant.md

```diff
@@ -303,6 +303,14 @@ Checking connectivity... done.
 Checking out files: 100% (1912/1912), done.`
 ```
 
+4. After cloning, [connect your fork to Zulip upstream with]
+(..git/cloning.html#step-1c-connect-your-fork-to-zulip-upstream)
+
+```
+git remote add upstream https://github.com/zulip/zulip.git
+git fetch upstream
+```
+
```

Comment:
this one should be combined with the above, like the rest.  This one is actually the most important, since it's the main page.